### PR TITLE
withclass27

### PR DESCRIPTION
**Pull request by kunireddisri**

### CODE REVIEW

File: force-app/main/default/classes/WiredActsController.cls [hunk 1]
```diff
@@ -11,14 +11,14 @@ public with sharing class WiredActsController {
        list<account> acts= [Select id,name,phone from account where name like :n];
        system.debug(acts); 
        return acts;      
-    }
```
Comment (sureshksv) [inline force-app/main/default/classes/WiredActsController.cls:14]: why there is a space

File: force-app/main/default/lwc/testingGit1/testingGit1.html [hunk 2]
```diff
@@ -1,4 +1,19 @@
 <template>
+    <lightning-card title="Wired Function Example" icon-name="custom:custom64">
+        <div >
+        
+            <template if:true={result}>
```
Comment (sureshksv) [inline force-app/main/default/lwc/testingGit1/testingGit1.html:5]: Please format properly

File: force-app/main/default/classes/WiredActsController.cls (hunk 1 above)
Comment (kunireddisri) [reply force-app/main/default/classes/WiredActsController.cls:14]: it came like that when i checked out that code from git

File: force-app/main/default/lwc/testingGit1/testingGit1.html (hunk 2 above)
Comment (kunireddisri) [reply force-app/main/default/lwc/testingGit1/testingGit1.html:5]: i will format in next commit

**Review by sureshksv — APPROVED** [review]

**Pull request merged**